content/md/products-super-power/product-associations.md

```diff
@@ -15,15 +15,25 @@ A product may be associated with none or several other products or product group
 This type of relationship enables you to define relationships like an alternative product or a substitution between products for example.
 :::
 
+:::info
+Since the 2.1, a product can be associated to a [product model](/articles/what-about-products-variants.html#what-is-a-product-model).
+:::
+
+:::warning
+Only products can have associations, [product models](/articles/what-about-products-variants.html#what-is-a-product-model) have no association.
```

Comment:
Can't be associated or can't have association?

Reply:
effectivement c'est ambigu